assets/js/app.js

```diff
@@ -1,4 +1,4 @@
-import Raven from "raven-js";
+import * as Sentry from '@sentry/browser';
```

Comment:
We need to use double quotes in js!

Reply:
![odio](https://cdnvxl.calciomercato.com/data/postImages/6363587c378816c6b8412ccbdb916c4ef058e573.png)